RELEASE-NOTES.md

```diff
@@ -104,24 +104,97 @@ buckets.default.allow_mult = true
+`"/usr/lib/riak-cs/lib/riak_cs-1.5.4/ebin"` to
+`"/usr/lib/riak-cs/lib/riak_cs-2.0.0/ebin"`.  ```
+
+If Riak-2.0-native way of configuration is preferred, 2. is the
+choice.  Copy all `riak_kv` configuration items of `app.config` to
+`advanced.config` as it is, with `add_paths` updated.
+
+This is because old style backend configuration same as 1.5 should be
+written in `riak_kv` section of `advanced.config`. See
 [Setting up the Proper Riak Backend](http://docs.basho.com/riakcs/1.5.4/cookbooks/configuration/Configuring-Riak/#Setting-up-the-Proper-Riak-Backend)
-for details.
+for details. `app.config` should be removed when `advanced.config`
+should be used.
+
+#### Redesign of memory sizing
+
+It is also worth noting that since LevelDB has changed its
+configuration item on memory size configuration, from `max_open_files`
+to `total_leveldb_mem_percent`. This specifies total amount of memory
+consumed by LevelDB. This configuration is very important because the
+default value of memory cap limitation has changed from proportional
+number of `max_open_files` to specifying percentage of system's
+physical memory size.
+
+Configuring `total_leveldb_mem_percent` is *strongly recommended* as
+its default value is
+[70%](https://github.com/basho/eleveldb/blob/2.0.2/priv/eleveldb.schema#L17),
+which might be too aggressive for multi-backend configuration that
+also uses bitcask. Bitcask keeps its keydir on memory, which could be
+fairly large depending on use cases.
+
+Hereby basic idea of redisigning memory sizing is described: estimate
+total number of objects with their size distribution. Total number of
+keys `N(b)` in Bitcask accross the whole cluster will be:
+
+```
+N(b) = N(o, size <= 1MB) + avg(o, size > 1MB)
+```
+
+where `N(o, size <= 1MB)` is number of objects whose size is less than
+1MB, and `avg(o, size > 1MB)` is average size of objects whose size is
+more than 1MB. If average lifetime of objects is comparably smaller
+than leeway period, take objects waiting for garbage collections into
+account as live objects on disk.
+
+Once numbers of keys figured out, estimate the amount of memory used
+by Bitcask keydir with
+[Bitcask Capacity Planning](http://docs.basho.com/riak/2.0.5/ops/building/planning/bitcask/).
 
-For later release than Riak 2.0.5, new configuration in Riak will be
-just
+Bucket size is always 19B (see `riak_cs_utils:to_bucket_name/2`), Key
+size is always 20B (see `riak_cs_lfs_utils:block_name/3`). Average
+Value Size might be 1MB if large objects are dominant, otherwise it
+should be estimated under specific usecase. Number of writes is 3.
+
+Note that this only takes bitcask into account, so another room of
+memory for LevelDB should be preserved as described above. Moreover,
+to improve IO performance, extra space for kernel disk cache would
+desirable:
+
+```
+Memory for storage = (Memory for backends) + (Memory for kernel cache)
+Memory for backends = (Memory for Bitcask) + (Memory for LevelDB)
```

Comment:
These relations are nice summary and seems to good start point for explanation.
How about putting them around the beginning of memory usage estimation / configuration?


1. These relations
2. Calculate bitcask keydir memory usage (because it is forced)
3. Determine leveldb percentage (and benchmark some configurations hopefully :see_no_evil: )